data/applications/pachyderm.yaml

```
@@ -6,12 +6,13 @@ spec:
   displayName: Pachyderm
   provider: Pachyderm
   description: |-
-    Pachyderm is a group of products that help data science teams automate and optimize their machine learning operations.
+      Pachyderm is the data foundation for machine learning.  
+      Pachyderm provides industry leading data versioning, pipelines and lineage that allow data science teams to automate the machine learning lifecycle and optimize their machine learning operations (MLOps).
```

Where will this section appear in the UI? I know each learning experience will get a card (with a title and description) on the RHODS Resources page, but I'm having trouble figuring out where this will appear.

---

This will be shown up in "Enabled" section and you can not see it unless you install Pachyderm operator in the cluster.

---

That makes sense, thank you! I was picturing the Resources page, so that is why I got confused haha. I left a few tweaks that help shorten is slightly and unify the 2 sentences a tad:

```suggestion
     Pachyderm is the data foundation for machine learning. It provides industry-leading data versioning, pipelines, and lineage for data science teams to automate the machine learning lifecycle and optimize machine learning operations (MLOps).
```